applications/DamApplication/CMakeLists.txt

```diff
@@ -2,131 +2,115 @@ set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
 message("**** configuring KratosDamApplication ****")
 
-include_directories( ${CMAKE_SOURCE_DIR}/kratos )
+################### PYBIND11
+include(pybind11Tools)
 
-#dependecies on other applications
+include_directories( ${CMAKE_SOURCE_DIR}/kratos )
```

Comment:
this should already be included

Reply:
What do you mean?

Reply:
I don't see anything wrong with that line either :S

Reply:
@RiccardoRossi ping! ;)

Reply:
i am just telling that i think it is included automatically

if i am wrong please approve and go ahead

Reply:
@roigcarlo could you approve this PR? ^_^'